compiler/setup.py

```diff
@@ -414,6 +414,20 @@ def find_git_submodule_revision(submodule_path):
 )
 print(f"Found compiler packages: {packages}")
 
+with open(
+    os.path.join(
+        IREE_SOURCE_DIR,
+        "compiler",
+        "bindings",
+        "python",
+        "iree",
+        "compiler",
+        "README.md",
+    ),
+    "rt",
+) as f:
+    README = f.read()
```

Comment:
A more lightweight solution could be to add `readme = "README.md"` to the `pyproject.toml` but as it only defines the build dependencies for this Python package, I would stick with what you got here.

Reply:
That could work. Note that we have README files in two locations for the runtime package, and I'm matching that structure here for the compiler package:

* https://github.com/iree-org/iree/blob/main/runtime/README.md (sibling to `setup.py` and `pyproject.toml`)
* https://github.com/iree-org/iree/blob/main/runtime/bindings/python/iree/runtime/README.md (content for PyPI)
* https://github.com/iree-org/iree/blob/main/compiler/README.md (sibling to `setup.py` and `pyproject.toml`)

Should probably make a pass through and clean up the files. Some were last updated halfway through code restructuring: https://github.com/iree-org/iree/blob/4477091bb04cf08e6468d895787b6cfd4fd2c8b7/runtime/README.md?plain=1#L1-L8

Reply:
Interesting idea. I feel like API docs and a package README are a bit different though :thinking:

Reply:
I have confused myself and therefore it is not an interesting idea at all and rather wrong. I thought the README I was referring to was used for RTD but that is obviously not the case you mentioned it is used for PyPi. We should than probably fuse the two READMEs we have for the runtime?